modules/ocf_hpc/files/slurmdbd.conf

```diff
@@ -0,0 +1,23 @@
+ArchiveEvents=yes
+ArchiveJobs=yes
+ArchiveResvs=yes
+ArchiveSteps=no
+ArchiveSuspend=no
+#ArchiveScript=/usr/sbin/slurm.dbd.archive
+#AuthInfo=/var/run/munge/munge.socket.2
+AuthType=auth/munge
+DbdHost=segfault
+DbdBackupHost=corruption
```

Comment:
if possible, that is

Reply:
That sounds like a good idea.

Reply:
Unfortunately, SLURM does not accept CNAMEs in configs, since it validates that the hostname specified in the config for the controller and DBD host match the hostname given by hostname -s